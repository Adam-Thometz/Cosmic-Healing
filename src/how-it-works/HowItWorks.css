.HowItWorks {
  display: flex;
  justify-content: space-between;
}

.HowItWorks-text-wrapper {
  text-align: left;
  padding: 0 0 0 6%;
  width: 30%;
}

.HowItWorks-img-wrapper {
  width: 70%;
}

.HowItWorks-img-wrapper img {
  width: 100%;
}