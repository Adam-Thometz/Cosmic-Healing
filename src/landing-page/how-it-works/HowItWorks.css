.HowItWorks {
  display: flex;
  justify-content: space-between;
  margin: var(--margin) 0;
  animation: slide-up var(--duration) calc(var(--delay) * var(--i)) both;
}

.HowItWorks-text-wrapper {
  text-align: left;
  padding: 0 7%;
  width: 50%;
}

.HowItWorks-step {
  margin: 36px 16px;
}

.HowItWorks-step::marker {
  font-size: 36px;
  font-family: Nimbus;
  white-space: pre-line;
}

.HowItWorks-step h3, .HowItWorks-step p {
  margin-left: 32px;
}

.HowItWorks-img-wrapper {
  width: 50%;
}

.HowItWorks-img-wrapper img {
  width: 100%;
}

.HowItWorks .Link {
  margin: 0 84px;
}

@media (max-width: 600px) {
  .HowItWorks {
    flex-direction: column-reverse;
  }
}