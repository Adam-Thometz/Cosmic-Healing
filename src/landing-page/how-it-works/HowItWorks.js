import React from 'react';

import './HowItWorks.css';

import Link from '../../_components/link/Link';
import SectionHeader from '../../_components/section-header/SectionHeader';
import SmallCaption from '../../_components/small-caption/SmallCaption';

import paperPlane from '../../_media/landing-page/paper-plane.png';

const HowItWorks = ({ style }) => {
  return (
    <section className='HowItWorks' style={style}>
      <aside className='HowItWorks-text-wrapper'>
        <SmallCaption text="HOW IT WORKS.." />
        <SectionHeader text="Start in three easy steps" />
        <ol>
          <li className='HowItWorks-step'>
            <SectionHeader small text="Browse services" />
            <p>Browse services and pick the right path for you.</p>
          </li>
          <li className='HowItWorks-step'>
            <SectionHeader small text="Get connected" />
            <p>Answer a few questions or schedule a free appointment and a clincian will reach out.</p>
          </li>
          <li className='HowItWorks-step'>
            <SectionHeader small text="Begin" />
            <p>Congratulations on taking the first step towards a rewarding experience. </p>
          </li>
        </ol>
        <Link primary>Get Started</Link>
      </aside>
      <aside className='HowItWorks-img-wrapper'>
        <img src={paperPlane} alt='' />
      </aside>
    </section>
  );
};

export default HowItWorks;