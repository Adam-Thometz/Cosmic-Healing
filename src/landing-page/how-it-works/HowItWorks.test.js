import React from "react";
import { render } from "@testing-library/react";
import HowItWorks from './HowItWorks';

describe('HowItWorks component', () => {
  it('renders without crashing', () => {
    render(<HowItWorks />);
  });

  it('matches the snapshot', () => {
    const { asFragment } = render(<HowItWorks />);
    expect(asFragment()).toMatchSnapshot();
  });
});