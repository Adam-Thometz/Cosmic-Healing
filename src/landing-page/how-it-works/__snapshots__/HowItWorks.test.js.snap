// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HowItWorks component matches the snapshot 1`] = `
<DocumentFragment>
  <section
    class="HowItWorks"
  >
    <aside
      class="HowItWorks-text-wrapper"
    >
      <small
        class="SmallCaption"
        style="color: black;"
      >
        HOW IT WORKS..
      </small>
      <h2
        class="SectionHeader"
      >
        Start in three easy steps
      </h2>
      <ol>
        <li
          class="HowItWorks-step"
        >
          <h3
            class="SectionHeader"
          >
            Browse services
          </h3>
          <p>
            Browse services and pick the right path for you.
          </p>
        </li>
        <li
          class="HowItWorks-step"
        >
          <h3
            class="SectionHeader"
          >
            Get connected
          </h3>
          <p>
            Answer a few questions or schedule a free appointment and a clincian will reach out.
          </p>
        </li>
        <li
          class="HowItWorks-step"
        >
          <h3
            class="SectionHeader"
          >
            Begin
          </h3>
          <p>
            Congratulations on taking the first step towards a rewarding and transformative experience. 
          </p>
        </li>
      </ol>
      <a
        class="Link primary"
      >
        Get Started
      </a>
    </aside>
    <aside
      class="HowItWorks-img-wrapper"
    >
      <img
        alt=""
        src="paper-plane.png"
      />
    </aside>
  </section>
</DocumentFragment>
`;
